<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forgotpasswd</class>
 <widget class="QDialog" name="Forgotpasswd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>140</y>
            <width>261</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>Enter your Email ID:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_email">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>160</y>
            <width>251</width>
            <height>25</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_chk">
          <property name="geometry">
           <rect>
            <x>310</x>
            <y>240</y>
            <width>89</width>
            <height>25</height>
           </rect>
          </property>
          <property name="text">
           <string>check</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QPushButton" name="pushButton_chk_1">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>210</y>
         <width>89</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>check</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_sq">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>120</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>+security Question</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_ans">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>120</y>
         <width>113</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>111</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>New Password</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Re-enter password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_pwd">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>181</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_pwd_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>210</y>
         <width>181</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_change">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>310</y>
         <width>151</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>Change password</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>210</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.906, y1:0.642, x2:0, y2:0.590909, stop:0 rgba(239, 41, 41, 255), stop:1 rgba(255, 255, 255, 255));
color: black;</string>
       </property>
       <property name="text">
        <string>password not matched</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
